examples/tfengine/modules/team.hcl

```diff
@@ -382,13 +391,10 @@ EOF
             with_state = "ANY"
           }
         }]
-        retention_policy = {
-          retention_period = 86400 # 1 day.
-        }
-        iam_members = [{
-          role   = "roles/storage.objectViewer"
-          member = "group:{{.prefix}}-data-viewers@{{.domain}}"
-        }]
+        # Skip for integration tests.
```

Comment:
Nit: I would rephrase slightly: "This is skipped for integration tests for <reason> [I assume so resources can be destroyed after], uncomment when writing a production template."

Reply:
Done